### DISCUSSION TITLE
[WASM + PWA + .NET 7] PWA is not working after deploying to Azure Static Web App, IIS, and GitHub Pages

### DISCUSSION THREAD

**Discussion by asadullah-refat-89** [unoplatform/uno#10929] [Q&A]
Hello @jeromelaban ,
I am trying to enable PWA for this repo: https://github.com/asadullahrifat89/honk-hero-uno-platform
This only has a WASM head and I have set these in the `.csproj`  file:

`<WasmShellIndexHtmlPath>index.html</WasmShellIndexHtmlPath>`
`<WasmPWAManifestFile>manifest.json</WasmPWAManifestFile>`

I have created the solution using the new solution template which asks for enabling PWA and thus I already had the `manifest.json` file added to the project.

The PWA installation prompt works if I run it locally in Visual Studio but when deploying it to Azure Static Web App, IIS, and GitHub Pages I get this error when the `Service Worker` is registering.
`Uncaught (in promise) TypeError: Failed to execute 'addAll' on 'Cache': Request failed`

I understand that some files mentioned in the cache are not reachable thus this error is popping up.

This used to work in .NET 6 by the way where I tried PWA in another project. But after that, I updated everything and am now targeting .NET 7 and finding it hard to make it work.

Does this have something to do with the scope and start_url of the manifest file? I don't know. Pls, help.

**Comment by jeromelaban** [comment]
Thanks for the repro! Let's track this in https://github.com/unoplatform/uno/issues/10944.

**Reply by asadullah-refat-89** [reply]
Alright.
